docs/tutorials/notebooks/Monge_Gap.ipynb

```diff
@@ -0,0 +1,734 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #53.        def neural_monge_gap(self) -&gt; Callable:
</pre><p>I think both <code>neural_monge_gap</code> and <code>neural_fitting_loss</code> properties add unnecessary layer of complexity.</p><p>I think they can be easily incorporated in the <code>loss_fn</code> inside <code>_get_step_fn</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>